src/components/pages/PointsDashboard/PointsDashboard.tsx

```diff
@@ -267,12 +271,10 @@ const UserPointInfo = () => {
                         <div className={'border-r border-white-10'}>
                             <div className='flex h-full w-full flex-col items-center justify-center overflow-hidden'>
                                 <span className='typography-caption-2 h-5 w-full text-center text-secondary7'>
-                                    Joined Date
+                                    Point Boost
                                 </span>
-                                <div className='typography-caption flex h-9 w-full items-center justify-center text-center text-white'>
-                                    {formatDate(
-                                        dayjs(userData?.user.joinedAt).unix()
-                                    )}
+                                <div className='typography-body-1 h-8 text-center text-md text-white'>
+                                    + {userData?.user.boostPercentage / 100}%
```

Comment:
can this become a non terminating decimal problem in JavaScript?
use percentFormat from formatNumbers.ts instead

Reply:
chromatic shows NaN